packages/frontend/package.json

```diff
@@ -79,22 +75,23 @@
         "monaco-sql-languages": "^0.12.2",
         "polished": "^4.2.2",
         "posthog-js": "^1.166.1",
-        "react": "^18.3.1",
+        "react": "^19.0.0",
         "react-ace": "^9.5.0",
         "react-countdown": "^2.3.5",
-        "react-dom": "^18.3.1",
+        "react-dom": "^19.0.0",
         "react-google-drive-picker": "^1.2.2",
-        "react-grid-layout": "^1.3.3",
+        "react-grid-layout": "1.3.4",
```

Comment:
The latest version has a bug https://github.com/react-grid-layout/react-grid-layout/issues/2083
So I pinned the version.